libs/remote_storage/src/s3_bucket.rs

```diff
@@ -67,6 +68,9 @@ impl S3Bucket {
             aws_config.bucket_name
         );
 
+        let provider_conf = ProviderConfig::without_region()
```

Comment:
This looks weird, we really need to build it without_region and the configure the region? Did not check if it had better api.

Reply:
I took this example from docs:
https://docs.rs/aws-config/0.56.0/aws_config/provider_config/struct.ProviderConfig.html#method.with_region

totally agree, that it looks strange

Reply:
Oki then, resolving.